In [1]:
#DEPENDENCIES & INITIALISATIONS
from IPython.display import HTML #For embedding videos
import math #Basic math functionality
import numpy as np #Numpy for working with arrays
import matplotlib.pyplot as plt #Plotting functionality 
#HTML('<center><iframe width="560" height="315" src="#" frameborder="0" allowfullscreen></iframe></center>')
print("- Dependencies imported 👍")
print("- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬")
print("- Author: Dr H Alwi & Dr SPCarroll, Sept 2020, Rev 0")

- Dependencies imported 👍
- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬
- Author: Dr H Alwi & Dr SPCarroll, Sept 2020, Rev 0


# Week 5: Curvilinear and Angular Motion
Reference: Chapter 13 of Engineering Mechanics - Statics and Dynamics Principles, Bedford A. & Fowler, W., 2003

## Question 1

The skier in the image below leaves the $20^{\circ}$ surface at $10\:m/s$. Determine the distance $d$ to the point where he lands. This is a tricky question - stick with it!

**Hints:**

- Establish an origin point at the skiers' location just before he departs the ramp.
- Determine expressions for his $x$ and $y$ velocity components and integrate these to get $x$ and $y$ position as a function of time. 
- Then think about how you can use these expressions, combined with the geometry of the slope (given for a reason) to determine the time the skier lands. 
- Once you know **when** he lands it's easy to work out his position at this time.
- Remember we're treating the skier as a projectile, so horizontal acceleration is zero and vertical acceleration is due to gravity.

![title](img/T2W5/img1.jpg)

**Solution 1**

First we need to determine the $x$ and $y$ components of velocity; since the magnitude is $10\:m/s$ and the take-off angle is $20^\circ$, we have,

\begin{align*}
v_x &= 10\cos 20^\circ\\
v_y &= -10\sin 20^\circ
\end{align*}

In [2]:
vMag = 10 #(m/s)
theta = 20 #(deg)
theta_rad = theta*math.pi/180
vx = 10*math.cos(theta_rad) #(m/s)
vy = -10*math.sin(theta_rad) #(m/s)
print('vx = {one} m/s'.format(one=round(vx,2)))
print('vy = {one} m/s'.format(one=round(vy,2)))

vx = 9.4 m/s
vy = -3.42 m/s


The $x$ component of acceleration, $a_x=0$, i.e. the velocity is constant therefore,

\begin{equation*}
v_x = \frac{\mathrm{d}x}{\mathrm{d}t} = 9.4\:m/s
\end{equation*}

Integrating,

\begin{equation*}
\int_0^x \mathrm{d}x = \int_0^t 9.4\mathrm{d}t
\end{equation*}

which yields,

\begin{equation*}
\boxed{
x=9.4t\:m
}
\end{equation*}

The $y$ component of acceleration is due to gravity,

\begin{equation*}
a_y = \frac{\mathrm{d}v_y}{\mathrm{d}t}=-9.81\:m/s^2
\end{equation*}

Therefore we need to integrate to determine $v_y$ as a function of time,

\begin{equation*}
\int_{-3.42}^{v_y} \mathrm{d}v_y = \int_0^t-9.81
\end{equation*}

\begin{equation*}
v_y = \frac{\mathrm{d}y}{\mathrm{d}t} = -3.42-9.81t
\end{equation*}

Integrating again to determine $y$ as a function of $t$,

\begin{equation*}
\int_0^y \mathrm{d}y = \int_0^t(-3.42-9.81t)\mathrm{d}t
\end{equation*}

which yields,

\begin{equation*}
\boxed{
y=-3.42t-4.905t^2\:m
}
\end{equation*}

The next step is to build an equation that describes the slope (inclined blue line) the skier will land on, see below. Ultimately we are trying to determine where the skiers' trajectory intersects with this line.

![title](img/T2W5/img4.jpg)

The slope of the line is,

\begin{equation*}
\text{slope}=\frac{\Delta x}{\Delta y} = \tan(45^\circ) = -1 (\text{negative because downwards})
\end{equation*}

So the equation of the line is,

\begin{equation*}
y=(-1)x + C
\end{equation*}

where $C$ is a constant. We determine $C$ using the fact that $x=0$ and $y=-3$, therefore,

\begin{align*}
-3&=(-1)0 + C\\
-3&=C
\end{align*}

Therefore the equation describing the $45^\circ$ slope is,

\begin{equation*}
\boxed{
y=-x-3\:m
}
\end{equation*}

We now have three simultaneous equations; we can obtain the time at which the skier lands by substituting equations for $x$ and $y$ into our equation describing the slope,

\begin{align*}
y &= -x-3\\
-3.42t-4.905t^2 &= -(9.4t)-3
\end{align*}

Solving for $t$ yields $t=1.6\:s$. We can now evaluate expressions for $x$ and $y$ at $t=1.6\:s$.

In [3]:
tLand = 1.6 #(s) Landing time
x = 9.4*tLand #(m)
y = -3.42*tLand - 4.905*tLand**2 #(m)
print('The distance travelled in the x direction is {one} m'.format(one=round(x,2)))
print('The distance travelled in the y direction is {one} m'.format(one=round(y,2)))

The distance travelled in the x direction is 15.04 m
The distance travelled in the y direction is -18.03 m


![title](img/T2W5/img5.jpg)

We can determine the distance $d$ as follows,

\begin{equation*}
d=\sqrt{15^2 + (18-3)^2}
\end{equation*}

In [4]:
d = (x**2 + (abs(18)-3)**2)**0.5
print('The distance d is {one} m'.format(one=round(d,2)))

The distance d is 21.24 m


## Question 2

The rotor of a jet engine is rotating at $10000\:rpm$ (revolutions per minute) when the fuel is cut off. The ensuing angular acceleration is $\alpha=-0.02\omega$, where $\omega$ is the angular velocity in $rad/s$. 

- (a) How long does it take the rotor to slowdown to $1000\:rpm$?
- (b) How many revolutions does the rotor turn while decelerating to $1000\:rpm?$

**Hint:** Remember to employ the same solution strategies you would in the case of linear motion. 

![title](img/T2W5/img6.jpg)

**Solution 2**

First things first, we need to work out the conversion between revolutions per minute and rads/sec,

\begin{equation*}
1\:rpm = 1\frac{\text{revolution}}{\text{min}}\times\frac{2\pi\:rad}{\text{revolution}}\times\frac{1\:\text{min}}{60\:s} = \frac{\pi}{30} rad/s 
\end{equation*}

(a) To determine the time taken for the velocity to reduce to $1000 \:rpm$ we need an equation for angular velocity as a function of time. Therefore starting with an expression for acceleration as a function of velocity, we separate variables and integrate,

\begin{equation*}
\frac{\mathrm{d}\omega}{\mathrm{d}t} = -0.02\omega
\end{equation*}

\begin{equation*}
\frac{\mathrm{d}\omega}{\omega} = -0.02\:\mathrm{d}t
\end{equation*}

Now integrating with velocity limits set at the initials and final velocity,

\begin{equation*}
\int_{10,000\pi/30}^{1000\pi/30}\frac{1}{\omega}\mathrm{d}\omega = \int_0^t-0.02\mathrm{d}t
\end{equation*}

Integrating and solving for $t$ yields,

\begin{equation*}
t = \frac{1}{0.02}\ln\left(\frac{10,000\frac{\pi}{30}}{1000\frac{\pi}{30}}\right)
\end{equation*}

In [5]:
t = (1/0.02)*math.log((10000*math.pi/30)/(1000*math.pi/30))
print('(a) The time taken to slow is {one} s'.format(one=round(t,1)))

(a) The time taken to slow is 115.1 s


(b) To determine the number of revolutions while decelerating we need an equation for angular displacement $\theta$, as a function of angular velocity $\omega$. Adopting the exact same strategy as for linear motion, we use the chain rule to separate variables,

\begin{equation*}
\frac{\mathrm{d}\omega}{\mathrm{d}t} = \frac{\mathrm{d}\omega}{\mathrm{d}\theta}\frac{\mathrm{d}\theta}{\mathrm{d}t} = \frac{\mathrm{d}\omega}{\mathrm{d}\theta}\omega = -0.02\omega
\end{equation*}

Therefore, 

\begin{equation*}
\mathrm{d}\omega = -0.02\:\mathrm{d}\theta
\end{equation*}

Integrating,

\begin{equation*}
\int_{10,000\pi/30}^{1000\pi/30}\mathrm{d}\omega - \int_0^\theta -0.02\:\mathrm{d}\theta
\end{equation*}

Evaluating the integral and solving for $\theta$,

\begin{equation*}
\theta =\frac{1}{0.02}\left[\left(10,000\frac{\pi}{30}\right)-\left(1000\frac{\pi}{30}\right)\right]
\end{equation*}

In [6]:
theta = (1/0.02)*((10000*math.pi/30)-(1000*math.pi/30)) #(rads)
print("Angular 'distance' covered is {one} radians".format(one=round(theta,2)))

Angular 'distance' covered is 47123.89 radians


Since $1\:\text{rev} = 2\pi$ radians,

In [7]:
revs = theta/(2*math.pi)
print('(b) The engine completes {one} revolutions while slowing down'.format(one=round(revs)))

(b) The engine completes 7500 revolutions while slowing down


## Question 3

A satellite is in a circular orbit of radius $R$ around the earth. Determine an expression for the velocity of the satellite. Note that the magnitude of the acceleration due to gravity at a distance $R$ from the centre of the earth is,

\begin{equation*}
g\frac{R_E^2}{R^2}
\end{equation*}

where $R_E$ is the radius of the earth.

![title](img/T2W5/img2.jpg)

**Solution 3**

We start by stating the expression for acceleration,

\begin{equation*}
a = \frac{\mathrm{d}v}{\mathrm{d}t}\mathbf{e}_t+\frac{v^2}{R}\mathbf{e}_n
\end{equation*}

This expression must equal the acceleration due to gravity towards the centre of the earth,

\begin{equation*}
\frac{\mathrm{d}v}{\mathrm{d}t}\mathbf{e}_t+\frac{v^2}{R}\mathbf{e}_n=g\frac{R_E^2}{R^2}\mathbf{e}_n
\end{equation*}

Note that the satellite doesn't have a tangential component of acceleration, only a normal component (gravity), from this we can conclude that the satellite's velocity along its circular orbital path is constant,

\begin{equation*}
\frac{\mathrm{d}v}{\mathrm{d}t} = 0
\end{equation*}

We simply need to equate the normal components of acceleration and solve for $v$,

\begin{equation*}
\boxed{
v = \sqrt{\frac{gR_E^2}{R}}
}
\end{equation*}

## Question 4

A centrifuge used to subject engineering components to high acceleration has a radius of $8 \:m$. It starts from rest at $t = 0$ seconds and during its two-minute acceleration phase it is programmed so that its angular acceleration $\alpha$ is given as a function of time in seconds,

\begin{equation*}
\alpha = 0.192 - 0.0016t\:rad/s
\end{equation*}

At $t = 120$ seconds, what is the magnitude of the acceleration a component is subjected to (positioned on the end of the $8m$ long centrifuge arm)?

![title](img/T2W5/img3.jpg)

**Solution 4**

To find the acceleration that a component will experience on the end of the $8\:m$ centrifuge arm, we need to determine the normal and tangential components of acceleration at $t=120$ seconds. 

We have derived expressions that relate these accelerations components $[a_n, \:a_t]$ to the angular velocity $\omega$ and angular acceleration $\alpha$,

\begin{equation*}
a_t=R\alpha
\end{equation*}

\begin{equation*}
a_n=R\omega^2
\end{equation*}

So our first step is to integrate our expression for $\alpha$ to determine $\omega$,

\begin{equation*}
\alpha = \frac{\mathrm{d}\omega}{\mathrm{d}t}=0.192-0.0016t
\end{equation*}

Separating variables and integrating,

\begin{equation*}
\int_0^{\omega} \mathrm{d}\omega= \int_0^{120} (0.192-0.0016t)\:\mathrm{d}t
\end{equation*}

Evaluating the integral yields,

\begin{equation*}
\omega = 0.192t - \frac{0.0016t^2}{2}\biggr\rvert_0^{120}
\end{equation*}

In [8]:
t=120 #(s)
omega=0.192*t-(0.5*0.0016*t**2) #(rad/s) Angular velocity 
print('The angular velocity is {one} rad/s'.format(one=round(omega,2)))

#Also need to determine alpha, angular acceleration at time t
alpha = 0.192 - 0.0016*t
print('The angular acceleration is {one} rad/s^2'.format(one=round(alpha,2)))

The angular velocity is 11.52 rad/s
The angular acceleration is 0.0 rad/s^2


Now we can determine the tangential and normal components of acceleration on the end of the $8\:m$ arm using the equations above,

In [9]:
R = 8 #(m)
at = R*alpha #(m/s^2) Tangential component of acceleration
an = R*omega**2 #(m/s^2) Normal component of acceleration

a = (at**2 + an**2)**0.5

print('The tangential component of acceleration is {one} m/s^2'.format(one=round(at,2)))
print('The normal component of acceleration is {one} m/s^2'.format(one=round(an,2)))
print('The total acceleration magnitude is {one} m/s^2'.format(one=round(a,2)))

The tangential component of acceleration is 0.0 m/s^2
The normal component of acceleration is 1061.68 m/s^2
The total acceleration magnitude is 1061.68 m/s^2


Note that the angular acceleration $\alpha$ has units of $rad/s$ while the acceleration components $[a_n, \:a_t]$ and their resultant have units of $m/s^2$. Be careful not to mix up these two quantities. Angular acceleration relates purely to the rate of change of angular velocity and is independent of the radius. This is distinct from the components of acceleration of a body traveling along a curved (circular in this case) path. 

## Answers

1. $21.24 \:m$
2. $115.1\:s$, $7500$
3. $v = \sqrt{\frac{gR_E^2}{R}}$
4. $1061.68 \:m/s^2$